package.json

```diff
@@ -56,7 +56,9 @@
     "demo:clean": "rimraf build/demo",
     "demo:css": "postcss src/static/css/styles.css -o src/static/css/tailwind.css",
     "demo:build": "rollup -c --silent --environment demoMode:true",
-    "docs": "node scripts/docs.js",
+    "docs": "run-s user-docs api-docs",
```

Comment:
⚠️ this means that the generate documentation workflow always build the api docs.
Is it what we want?